runtime/compiler/p/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -801,6 +802,35 @@ uint8_t *J9::Power::AheadOfTimeCompile::initializeAOTRelocationHeader(TR::Iterat
          }
          break;
 
+      case TR_DebugCounter:
+         {
+         TR::DebugCounterBase *counter = (TR::DebugCounterBase *) relocation->getTargetAddress();
+         if (!counter || !counter->getReloData() || !counter->getName())
+            comp->failCompilation<TR::CompilationException>("Failed to generate debug counter relo data");
+
+         TR::DebugCounterReloData *counterReloData = counter->getReloData();
+
+         UDATA offset = fej9->sharedCache()->rememberDebugCounterName(counter->getName());
```

Comment:
I would prefer this to be a JIT data type rather than UDATA, just for consistency. I know it doesn't architecturally matter any more now that we're in the same repo, but I think the consistency is still helpful until we get around to unifying all our types.